specification/batch/data-plane/Microsoft.Batch/stable/2018-12-01.8.0/BatchService.json

```diff
@@ -15017,8 +15024,8 @@
           "items": {
             "$ref": "#/definitions/ApplicationPackageReference"
           },
-          "title": "A list of application packages to be installed on each compute node in the pool.",
-          "description": "Changes to application package references affect all new compute nodes joining the pool, but do not affect compute nodes that are already in the pool until they are rebooted or reimaged. If this element is present, it replaces any existing application package references. If you specify an empty collection, then all application package references are removed from the pool. If omitted, any existing application package references are left unchanged."
+          "title": "The list of application packages to be installed on each compute node in the pool.",
+          "description": "The list replaces any existing application package references on the pool. Changes to application packages affect all new compute nodes joining the pool, but do not affect compute nodes that are already in the pool until they are rebooted or reimaged. There can be a maximum of 10 application package references on any given pool. If omitted, any existing application package references are left unchanged."
```

Comment:
If you are changing 'Changes to application package references' => 'Changes to application packages' also apply to 'The list replaces any existing application package references on the pool.' and other references in the body of text